plugins/inputs/nfsclient/README.md

```diff
@@ -30,20 +64,14 @@ Always collected:
     - write_exe (integer, miliseconds) - The number of miliseconds it took to process the RPC WRITE operations.
     - write_rtt (integer, miliseconds) - The rount-trip time for RPC WRITE operations.
 
-In addition enabling `fullstat` will make many more metrics available.  Please see the References for explanations of the the specific fields.
-See References for more details.
+In addition enabling `fullstat` will make many more metrics available.
 
 #### Tags
 
 - All measurements have the following tags:
     - mountpoint The local mountpoint, for instance: "/var/www"
     - serverexport The full server export, for instance: "nfsserver.example.org:/export"
 
-#### References
-1. [nfsiostat](http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=summary)
-2. [net/sunrpc/stats.c - Linux source code](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/net/sunrpc/stats.c)
-3. [What is in /proc/self/mountstats for NFS mounts: an introduction](https://utcc.utoronto.ca/~cks/space/blog/linux/NFSMountstatsIndex)
-4. [The xprt: data for NFS mounts in /proc/self/mountstats](https://utcc.utoronto.ca/~cks/space/blog/linux/NFSMountstatsXprt)
```

Comment:
I see at least 3 still referenced in the text. We should keep the references or change the code to not cite [3].

Reply:
Sorry, what?  Collecting accurate information about Linux NFS details is non-trivial.  Someone in the future is going to have reverse-engineer both the kernel /proc output and this plugin.  Having the references in the README as documentation is important.  Why would you want to remove them?

Reply:
Hey no offence intented! I just noticed that the citation is not referenced in the text! I see your point and agree that keeping it is sensible.